.npmignore

```diff
@@ -0,0 +1 @@
+# Empty npmignore to avoid ignoring js files when publishing
```

Comment:
Usually I'd say to ignore the source coffeescript files and tests and publish just the runtime files, but it doesn't make much difference in this repo